cou/logging.py

```diff
@@ -78,8 +89,6 @@ def setup_logging(log_level: str = "INFO") -> None:
     root_logger.addHandler(log_file_handler)
     root_logger.addHandler(console_handler)
 
-    progress_indicator.stop_and_persist(text=f"Full execution log: '{file_name}'")
```

Comment:
Can we retain printing the log file path at the beginning too?  Then the user can see the log file path to find it while cou is still running if they desire.

Reply:
Ok, nice idea.